styles.css

```diff
@@ -428,3 +432,110 @@ header {
 .about-li-category {
   list-style-type: none;
 }
+
+.contact-section {
+  background-color: #6070ff;
+  background-image: url('img/contact\ form\ shapes-mobile.svg');
+  background-position: right;
+  background-repeat: no-repeat;
+  border-top-left-radius: 80px;
+  height: 860px;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+}
+
+.contact {
+  padding-top: 2cm;
+  padding-left: 1.5cm;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  color: #fff;
+  margin-bottom: 10px;
+}
+
+.contact-text {
+  width: 327px;
+  height: 140px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact-form {
+  width: 325px;
+}
+
+.name {
+  border-radius: 5px;
+  border-color: transparent;
+  height: 48px;
+  width: 298px;
+  margin: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  align-items: center;
+  color: #172b4d;
+  padding-left: 10px;
+}
+
+.email {
+  border-radius: 5px;
+  border-color: transparent;
+  height: 48px;
+  width: 298px;
+  margin: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: 'normal';
+  font-weight: 200;
+  align-items: center;
+  color: #172b4d;
+  margin-left: 10px;
+  margin-right: 10px;
+  padding-left: 10px;
+}
+
+.text-box {
+  border-radius: 5px;
+  border-color: transparent;
+  height: 170px;
+  width: 298px;
+  margin: 10px;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  align-items: center;
+  color: #172b4d;
+  padding: 10px;
+}
+
+.button-contact {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```